src/main/features/claimant-response/routes/confirmation.ts

```diff
@@ -0,0 +1,22 @@
+import * as express from 'express'
+
+import { Paths } from 'claimant-response/paths'
+import { Claim } from 'claims/models/claim'
+import { ErrorHandling } from 'shared/errorHandling'
+import { getRepaymentPlanOrigin } from 'claimant-response/helpers/settlementHelper'
+import { MomentFactory } from 'shared/momentFactory'
+
+/* tslint:disable:no-default-export */
+export default express.Router()
+  .get(Paths.confirmationPage.uri,
+    ErrorHandling.apply(async (req: express.Request, res: express.Response, next: express.NextFunction) => {
+      const claim: Claim = res.locals.claim
+
+      res.render(
+        Paths.confirmationPage.associatedView,
+        {
+          claim: claim,
+          confirmationDate: MomentFactory.currentDate(),
```

Comment:
this needs to come from the claim otherwise someone can refresh it the next day and they'll get a different date

Reply:
We are not storing any date for this in claim atm, and I have been told this is current date. I hope there would be some guard to protect re-visiting the same confirmation page.

Reply:
@geek-soft can we revisit the design?

Reply:
Had a discussion with @geek-soft and we agreed to introduce date time for the party statements. but we will do that in separate story as we have priority for front-end now. I will be creating a new story for this.